src/_guides/language/evolution.md

```diff
@@ -390,20 +415,19 @@ For example:
 
 ```dart
 // Description of what's in this file.
-// @dart = 2.7
+// @dart = 2.17
 import 'dart:math';
 ...
 ```
 
-The `@dart` string must be in a `//` comment
-(not `///` or `/*`),
+The `@dart` string must be in a `//` comment (not `///` or `/*`),
 and it must appear before any Dart code in the file.
 Whitespace (tabs and spaces) doesn't matter,
 except within the `@dart` and version strings.
-As the example above shows,
+As the previous example shows,
 other comments can appear before the `@dart` comment.
 
-For more information about how language versioning works, see the
+To learn how language versioning works, check out the
```

Comment:
Isn't this whole section about how language versioning works?

We could either drop this or be more specific about why they would want to visit the specification.

Reply:
**comment:** Updated.